tests/test_azureplugin.py

```diff
@@ -277,12 +298,16 @@ def _get_no_nameservers_resolver():
 
 
 # ----------------------------------------------------------------------------
-def _get_proper_extensions_response():
+def _get_proper_extensions_response(upper = False):
     """Return a response that matches extensions config"""
     response = Response()
+    region = 'useast'
+    if upper:
+        # test region hint from MSFT are treated in lowercase
+        region = region.upper()
     response.status_code = 200
     data = 'the_doc '
-    data += '<Location>useast</Location>'
+    data += f"<Location>{region}</Location>"
```

Comment:
This will break the tests on SLE 12 which has Python 3.4 and does not support format strings. I'll fix it in a follow on PR. "Cosmetic" changes like this are really unnecssary.